code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -188,8 +195,9 @@
 	return COMSIG_SABOTEUR_SUCCESS
 
 /obj/item/kinetic_crusher/update_icon_state()
-	inhand_icon_state = "crusher[HAS_TRAIT(src, TRAIT_WIELDED)]" // this is not icon_state and not supported by 2hcomponent
-	return ..()
+	if(!override_twohandedsprite) // Another zubber edit hee hoo
+		inhand_icon_state = "crusher[HAS_TRAIT(src, TRAIT_WIELDED)]" // this is not icon_state and not supported by 2hcomponent
+		return ..()
```

Comment:
This can be made more modular and this isnt needed if it is handled by the modular file